--- clients/apps/browser/src/autofill/browser/main-context-menu-handler.ts	2024-05-06 05:04:23.104119887 +0000
+++ main-context-menu-handler.ts	2024-05-06 05:05:16.128119338 +0000
@@ -72,7 +72,7 @@
       id: COPY_VERIFICATION_CODE_ID,
       parentId: ROOT_ID,
       title: this.i18nService.t("copyVerificationCode"),
-      checkPremiumAccess: true,
+      // checkPremiumAccess: true,
     },
     {
       id: SEPARATOR_ID + 1,
